lib/src/r_tree/node.dart

```diff
@@ -16,6 +16,8 @@
 
 part of r_tree;
 
+const noMBR = const Rectangle(0, 0, 0, 0);
```

Comment:
`const` is redundant in this context
```suggestion
const noMBR = Rectangle(0, 0, 0, 0);
```